R/lint.R

```diff
@@ -231,17 +241,23 @@ lint_dir <- function(path = ".", relative_path = TRUE, ..., exclusions = NULL,
 #'   lint_package(
 #'     linters = with_defaults(semicolon_linter = semicolon_terminator_linter())
 #'     cache = TRUE,
-#'     exclusions = list("inst/doc/creating_linters.R" = 1, "inst/example/bad.R")
+#'     exclusions = list("inst/doc/creating_linters.R" = 1, "inst/example/bad.R", "data")
```

Comment:
This code doesn't work at present. Could you revert the comment back to it's original form.

Reply:
I'm a bit surprised that this doesn't work, but since we're probably touching this in #580 anyway, I'll revert to the old docs for now instead of fixing.